docs/umberto.json

```
@@ -13,6 +13,9 @@
 			"**/index.html"
 		]
 	},
+	"permalinkDisableDomains": [
+		"ckeditor5.github.io"
```

```suggestion
		"ckeditor5.github.io",
		"localhost",
		"127.0.0.1"
```

To cover local envs too.

---

Please add also `fake.ckeditor.com` to excluded domains.